src/utils/date.ts

```diff
@@ -0,0 +1,24 @@
+import dayjs from 'dayjs';
+
+export const dateFormatMonth = 'MMMM YYYY';
```

Comment:
You're not using this constant anywhere, so maybe there is no need for exporting it?

Reply:
Good catch 🦅 👁️